src/metabolomics_spectrum_resolver/cli.py

```diff
@@ -0,0 +1,13 @@
+import click
+
+
+@click.command()
+@click.option('--port', help='The port, defaults to 5000')
+@click.option('--host', help='The server, defaults to localhost. Switch to 0.0.0.0 to make externally available')
+def main(port, host):
+    from metabolomics_spectrum_resolver.app import app
```

Comment:
import happens inside function for faster loading times if you do `metabolomics_spectrum_resolver --help` from the shell